### ISSUE TITLE
[Accessibility] Make terminal a11y buffer even more accessible

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#15568]
CC -meganrogge -Tyriar -isidorn

I have tested the terminal a11y buffer by building the latest vscode main branch on my end.

Good news is that the buffer is now readable  Thank you very much! I am sharing some suggestions along with bug reports.

- [ ] No clue on whether users have reached the end of the terminal buffer.

Say, you are navigating the a11y terminal buffer via UpArrow/DownArrow. When there are repetitive prompt line, such as "PS C:\Users\john>" screen reader users cannot easily understand whether they have reached the end of the buffer or that's just the repetitive prompt message.

I suggest VSCode team add audio cues to indicate the buffer's top/bottom ending points. Instead of making a new sound, I personally think a gentle chime audio cue like "Terminal Bell (Disabled)" could be used for this purpose.

- [ ] Terminal a11y buffer is not read-only.

Since this is contenteditable textbox, unintended delete key can mess up the output content. I confirmed that delete key works inside the buffer.

It would be even nicer if this buffer could be marked up with readonly to prevent users from accidentally deleting their output content.

- [ ] Terminal live output is not announced.

As far as I tested on Windows with JAWS and NVDA, terminal aria-live event is not auto-announced. I typed "echo hello" in terminal, but there was no read-aloud feedback.

- [ ] Ctrl+UpArrow/Ctrl+DownArrow still does not trigger line-by-line announcement.

This issue does not seem to be fixed on Windows.
